src/strings/binary_length.jl

```diff
@@ -0,0 +1,28 @@
+#=
+This algorithm features use of the OEIS entry A070939 - 
+  Length of Binary Representation of n.
+
+The function, as believed, is O(n) 
+=#
+
+function binLength()  
+  println("Enter Maximum Number Please")
+  maxNum = parse(UInt64, readline())
+  
+  mod = 0
+  len = 1
+  arr = Array{Int64}(undef, maxNum, 1)
+   
+  Threads.@threads for i in 1:maxNum

```

Comment:
I am not quite sure why `Threads.@threads` is needed here. I tested your code with `BencmarkTools` both with unthreaded and threaded.

```julia
julia> @benchmark binary_length("10")
BenchmarkTools.Trial: 10000 samples with 10 evaluations.
 Range (min … max):  1.912 μs … 313.085 μs  ┊ GC (min … max): 0.00% … 98.35%
 Time  (median):     2.117 μs               ┊ GC (median):    0.00%
 Time  (mean ± σ):   2.223 μs ±   3.139 μs  ┊ GC (mean ± σ):  1.39% ±  0.98%

   █▄ ▄▃▂                                                      
  ████████▆▅▃▃▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▁▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂▂ ▃
  1.91 μs         Histogram: frequency by time         4.5 μs <

 Memory estimate: 736 bytes, allocs estimate: 9.

julia> @benchmark binary_length_unthreaded("10")
BenchmarkTools.Trial: 10000 samples with 986 evaluations.
 Range (min … max):  51.660 ns … 628.397 ns  ┊ GC (min … max): 0.00% … 90.84%
 Time  (median):     54.382 ns               ┊ GC (median):    0.00%
 Time  (mean ± σ):   59.278 ns ±  28.942 ns  ┊ GC (mean ± σ):  2.52% ±  4.93%

  ▅▇█▇▅▄▃▂▁▁▂▃▃▂▂                                              ▂
  ████████████████▆▆▆▅▇▇▇▅▆▅▅▆▆▇██▇███▇█▇▇▆▅▆▅▆▆▆▅▆▆▅▅▅▄▅▄▅▄▄▅ █
  51.7 ns       Histogram: log(frequency) by time       109 ns <

 Memory estimate: 144 bytes, allocs estimate: 1.
 ```
Though this benchmark is not something *we* should actually care about. I just want to know why we use `Threads.@threads` here.